app/views/schools/advice/electricity_recent_changes/_insights.html.erb

```diff
@@ -5,4 +5,4 @@
 <%= render 'schools/advice/section_title', section_id: 'your_recent_electricity_use', section_title: t('advice_pages.electricity_recent_changes.insights.your_recent_electricity_use.title') %>
 <%= render 'your_recent_electricity_use_table' %>
 <%= render 'schools/advice/section_title', section_id: 'how_do_you_compare', section_title: t('advice_pages.electricity_recent_changes.insights.how_do_you_compare.title') %>
-<%= t('advice_pages.electricity_recent_changes.insights.how_do_you_compare.summary_html', comparison_link: benchmark_path(benchmark_type: :change_in_electricity_consumption_recent_school_weeks)) %>
+<%= t('advice_pages.electricity_recent_changes.insights.how_do_you_compare.summary_html', comparison_link: benchmark_path(benchmark_type: :change_in_electricity_consumption_recent_school_weeks, benchmark: { school_group_ids: [@school.school_group_id] })) %>
```

Comment:
There's a helper for this: `benchmark_for_school_group_path` just need to provide the benchmark, could you update to use that?

Reply:
:+1: I've pushed an update to use the helper.